tests/userManagement.spec.ts

```diff
@@ -1,19 +1,18 @@
-import {expect, test} from '@playwright/test';
+import {test} from '@playwright/test';
 
 import {AccountPageFlow} from '@_flow/accountPage.flow';
 import {CommonFlow} from '@_flow/common.flow';
 import {LoginPageFlow} from '@_flow/loginPage.flow';
 import {RegisterPageFlow} from '@_flow/registerPage.flow';
-import {LoginPage} from '@_pages/login.page';
-import {RegisterPage} from '@_pages/register.page';
-import envConfig from '@_src/config/envConfig';
-import {generateUserData} from '@_src/helpers/generateUserData.helper';
+import {UserDataFactory} from '@_src/factory/generateUserData.factory';
+import {authenticateAndGetBearerToken, checkIfUserExistsByID, deleteUserWithAPI} from '@_src/helpers/api.helper';
 
-test.describe('User Management', () => {
-    const userData = generateUserData();
+test.describe('User Management', {tag: '@userManagement'}, () => {
+    const userDataFactory = new UserDataFactory();
+    const userData = userDataFactory.generateUserData();
+    let userID;
 
```

Comment:
Make sure to include `test.afterAll` to clean up the user if `userID` is defined. This ensures proper test environment cleanup after the test execution and allows for the user to be deleted if the test sequence is not fully executed.